io.vikunja.Desktop.appdata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.vikunja.Desktop</id>
+  <name>Vikunja</name>
+  <summary>The open-source, self-hostable to-do app. Organize everything, on all platforms.</summary>
+  <project_license>AGPLv3</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">io.vikunja.Desktop</launchable>
+  <content_rating type="oars-1.1" />
+  <url type="homepage">https://vikunja.io</url>
+  <url type="bugtracker">https://kolaente.dev/vikunja/desktop/issues</url>
+  <url type="help">https://community.vikunja.io</url>
+  <releases>
+    <release version="0.21.0" date="2023-7-10" />
+  </releases>
+</component>
```

Comment:
missing `screenshots` and a proper `description`

Reply:
Is there any reference on how I should provide these?

Reply:
From the documentatin you acknowledged having read. https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/

Reply:
Well the links I acknowledged didn't exactly link back to the article you mentioned. But that's nitpicking.